pyproject.toml

```diff
@@ -1,3 +1,13 @@
+[build-system]
```

Comment:
Please undo this change

Reply:
I agree, but setting up build-system was the only way I found to prevent the error described on #49

Reply:
When building from source these depends need to be installed on the isolated build environment. It is impossible to execute `setup.py` since the dependencies will only be installed after importing the function `version_from_git`

Reply:
I opened another PR that should reproduce this problem: #51

Reply:
Also note I didn't include all dependencies. Only the ones required for setup.
This is equivalent to `setup_requires` on `setup.py` (but this wouldn't solve the problem because setup.py would break before setting build dependencies :shrug: